src/api/Intelligence.js

```diff
@@ -68,22 +67,26 @@ class Intelligence extends Base {
     async extractStructured(request: AiExtractStructured): Promise<AiExtractResponse> {
         this.errorCode = ERROR_CODE_EXTRACT_STRUCTURED;
 
-        const url = `${this.getBaseApiUrl()}/ai/extract_structured`;
+        const { items } = request;
+        if (!items || items.length === 0) {
+            throw new Error('Missing items!');
+        }
 
-        let suggestionsResponse = {};
-        try {
-            suggestionsResponse = await this.xhr.post({
-                url,
-                data: request,
-                id: `file_${request.items[0].id}`,
-            });
-        } catch (e) {
-            const { status } = e;
-            if (isUserCorrectableError(status)) {
-                throw e;
+        items.forEach(item => {
```

Comment:
Do we need a forEach here? I see we only use items[0] for the `suggestionsResponse`. If so, can we just throw an error only if `!items[0].id`?